package.json

```diff
@@ -23,16 +26,12 @@
       "react-app/jest"
     ]
   },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
+  "browserslist": [
```

Comment:
Please revert the changes to the "browserslist" key

Reply:
This was causing problems for Map to display. It was the only solution to get rid of the errors. 
The development part "last 1 chrome version", .... was actually the problem so we had to eliminate it.

Reply:
Done

Reply:
Okay, that seems the problem
I guess I have to revert it again cuz now it's breaking :/

Reply:
> Okay, that seems the problem I guess I have to revert it again cuz now it's breaking :/

LGTM

Reply:
> LGTM

It might be your `node_modules` cache 🤔

Reply:
You have to delete  .cache file of node_modules.  Then it'll work again.